internal/server/data/data.go

```diff
@@ -41,7 +41,30 @@ func NewDB(connection gorm.Dialector, loadDBKey func(db *gorm.DB) error) (*gorm.
 		return nil, fmt.Errorf("migration failed: %w", err)
 	}
 
-	return db, nil
+	// TODO: initialize, and populate settings and org on DB
+	dataDB := &DB{DB: db}
+	if err := initialize(dataDB); err != nil {
+		return nil, fmt.Errorf("initialize database: %w", err)
+	}
+
+	return dataDB, nil
+}
+
+// DB wraps the underlying database and provides access to the default org,
+// and settings.
+type DB struct {
```

Comment:
I need this type for removing Gorm, and it helps us here as well by giving us a way to return the settings and default org.